backend/src/test/java/com/woowacourse/levellog/presentation/PreQuestionControllerTest.java

```diff
@@ -0,0 +1,403 @@
+                    .andExpect(jsonPath("message")
+                            .value("입력한 levellogId와 사전 질문의 levellogId가 다릅니다. 입력한 levellogId : 1"));
+
+            // docs
+            perform.andDo(document("pre-question/delete/exception/wrong-levellog"));
+        }
+
+        @Test
+        @DisplayName("저장되어있지 않은 사전 질문을 삭제하는 경우 예외를 던진다.")
+        void deleteById_PreQuestionNotFound_Exception() throws Exception {
+            // given
+            given(jwtTokenProvider.getPayload(TOKEN)).willReturn("4");
+            given(jwtTokenProvider.validateToken(TOKEN)).willReturn(true);
+
+            BDDMockito.willThrow(new PreQuestionNotFoundException("작성한 사전 질문이 존재하지 않습니다."))
+                    .given(preQuestionService)
+                    .deleteById(1L, 1L, 4L);
+
+            // when
+            final ResultActions perform = requestDelete("/api/levellogs/1/pre-questions/1", TOKEN);
+
+            // then
+            perform.andExpect(status().isNotFound())
+                    .andExpect(jsonPath("message")
+                            .value("사전 질문이 존재하지 않습니다."));
+
+            // docs
+            perform.andDo(document("pre-question/delete/exception/notfound"));
+        }
+
+        @Test
+        @DisplayName("타인의 사전 질문을 삭제하는 경우 예외를 던진다.")
+        void deleteById_FromNotMyPreQuestion_Exception() throws Exception {
+            // given
+            given(jwtTokenProvider.getPayload(TOKEN)).willReturn("4");
+            given(jwtTokenProvider.validateToken(TOKEN)).willReturn(true);
+
+            BDDMockito.willThrow(new UnauthorizedException("자신의 사전 질문이 아닙니다."))
+                    .given(preQuestionService)
+                    .deleteById(1L, 1L, 4L);
+
+            // when
+            final ResultActions perform = requestDelete("/api/levellogs/1/pre-questions/1", TOKEN);
+
+            // then
+            perform.andExpect(status().isUnauthorized())
+                    .andExpect(jsonPath("message")
+                            .value("권한이 없습니다."));
+
+            // docs
+            perform.andDo(document("pre-question/delete/exception/not-my-pre-question"));
+        }
+    }
+
+    private ResultActions requestPost(final String url, final String token, final String requestContent)
+            throws Exception {
+        return mockMvc.perform(post(url)
+                        .header(HttpHeaders.AUTHORIZATION, "Bearer " + token)
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(requestContent))
+                .andDo(print());
+    }
+
+    private ResultActions requestUpdate(final String url, final String token, final String requestContent)
+            throws Exception {
+        return mockMvc.perform(put(url)
+                        .header(HttpHeaders.AUTHORIZATION, "Bearer " + token)
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(requestContent))
+                .andDo(print());
+    }
```

Comment:
```suggestion
    private ResultActions requestPut(final String url, final String token, final String requestContent)
            throws Exception {
        return mockMvc.perform(put(url)
                        .header(HttpHeaders.AUTHORIZATION, "Bearer " + token)
                        .contentType(MediaType.APPLICATION_JSON)
                        .content(requestContent))
                .andDo(print());
    }
```
메서드로 잘 추출해주셨군요!!!!!!!!!!!!!!!!!!!!! 💯
requestXXX 메서드 이름을 http method이름을 따라가는게 명확하지 않을까요? 테스트에서만 쓰이는 메서드니까요 😉